File: question4.c
//WAP to count digits in a number
#include<stdio.h>
int main(){
    int num,count=0;
    printf("enter the no. to count digit");
    scanf("%d",&num);
    while(num>0){
    num=num/10;
    count++ ; 
    }
    printf("%d",count);
    return 0;
}

File: java_assign/DAY1/QUESTION4.java
//to count digit in a no.
package DAY1;
import java.util.Scanner;
public class QUESTION4 {
    public static void main(String[] args) {
        Scanner Sc =new Scanner(System.in);
        System.out.println("enter no. to count digit");
        int num = Sc.nextInt();
        int count =0;
        while(num>0){
            num=num/10;
            count++;
        }
        System.out.println("no. of digits in entered no." + count);
        
    }
    
}
